config/config_test.go

```diff
@@ -38,6 +39,27 @@ func TestConfig(t *testing.T) {
 						DefaultRequestTimeout:        10 * time.Second,
 						CantabularHealthcheckEnabled: false,
 					},
+					MetadataCollection: "filterMetadata",
+					Mongo: mongodb.MongoDriverConfig{
+						Username:        "",
+						Password:        "",
+						ClusterEndpoint: "localhost:27017",
+						Database:        "filters",
+						Collections: map[string]string{
+							"filterMetadata": "filterMetadata",
```

Comment:
Hmm we're not really storing metadata about a filter, it's metadata about the population-type.. I wonder what a better name/place for the collection would be

Reply:
`defaultDatasetMetadata`